source/getting-started/core-concepts.md

```diff
@@ -1,58 +1,136 @@
-To get started with Ember.js, there are a few core concepts you
-should understand.
+Before moving to more focused topics, it is important to review several
+high-level Ember.js concepts.
 
 ## Templates
 
-Templates, written in the Handlebars language, describe the user interface of
-your application. In addition to plain HTML, templates can contain expressions,
-like `{{title}}` or `{{author}}`, which take information from a component or
-controller and put it into HTML. They can also contain helpers, like
-`{{if isAdmin}}30 people have viewed your blog today.{{/if}}.` Finally, they can
-contain components, such as if a template listing out all the blog posts in an
-app rendered a component for each post.
+Ember.js uses templates to organize the layout of HTML in an application. Most
```

Comment:
There are places where "Ember" is used and also places where "Ember.js" is used throughout this (and the rest of the guides). Maybe we should decide which one to use going forward to be consistent?